### PR TITLE
Initial features

### PR DESCRIPTION
**Pull request by leightonthomas**
Added initial features

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "minimum-stability": "dev",
+  "prefer-stable": true,
+  "require": {
+    "php": ">=5.3.2",
+    "symfony/framework-bundle": "~2.3",
+    "jmikola/wildcard-event-dispatcher-bundle": "^1.0",
+    "monolog/monolog": "^1.22"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "~3.7.0"
```
Comment (Brunty) [inline composer.json]: Could we up the version of phpunit to `^5.7`?

File: src/Vivait/LoggingBundle/Resources/config/config.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+parameters:
+    vivait_logging_app_name: AcmeApp
```
Comment (Brunty) [inline src/Vivait/LoggingBundle/Resources/config/config.yml]: Is this needed anymore?

Comment (leightonthomas) [inline src/Vivait/LoggingBundle/Resources/config/config.yml]: Removed in the next commit

File: composer.json (hunk 1 above)
Comment (leightonthomas) [inline composer.json]: Changed in next commit

**Pull request merged**